static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -390,6 +394,13 @@ export class UsageChart extends Component<Props, State> {
         name: SeriesTypes.PROJECTED,
       });
     }
+
+    if (chartSeries) {
+      chartSeries.forEach(chartOption => {
+        legend.push({name: `${chartOption.name}`});
+      });
+    }
```

Comment:
When we add the spike protection data, we want to ensure we can pass a name to the series as well

Reply:
@leeandher This adds `Quota Line` to the chart legend unexpectedly. Could it be optional per `chartOption`?